src/main/resources/assets/tconstruct/lang/zh_cn.json

```
@@ -2162,19 +2296,22 @@
   "modifier.tconstruct.ductile.description": "工具更加耐用，采掘速度更快，伤害更高",
   "modifier.tconstruct.maintained": "持久",
   "modifier.tconstruct.maintained.flavor": "永葆青春，永保锋利！",
-  "modifier.tconstruct.maintained.description": "耐久值越高，工具的采掘速度越快",
-  "modifier.tconstruct.maintained.mining_speed": "持久采掘",
+  "modifier.tconstruct.maintained.description": "耐久度越高，工具的采掘速度越快",
+  "modifier.tconstruct.maintained.mining_speed": "持久采掘速度",
+  "modifier.tconstruct.maintained.velocity": "持久初速度",
   "modifier.tconstruct.maintained_2": "持久",
   "modifier.tconstruct.maintained_2.flavor": "更简易的磨砺方式！",
-  "modifier.tconstruct.maintained_2.description": "耐久值越高，工具的采掘速度越快",
+  "modifier.tconstruct.maintained_2.description": "耐久度越高，工具的采掘速度越快",
   "modifier.tconstruct.temperate": "恒温",
   "modifier.tconstruct.temperate.flavor": "极端环境开采作业！",
   "modifier.tconstruct.temperate.description": "工具在极端温度下加快采掘速度",
-  "modifier.tconstruct.temperate.mining_speed": "恒温速度",
+  "modifier.tconstruct.temperate.speed": "恒温速度",
+  "modifier.tconstruct.temperate.reinforced": "恒温Reinforcement",
```

```suggestion
  "modifier.tconstruct.temperate.reinforced": "恒温加固板",
```
好像……直译是这样的（匠魂里面刚好有这个）

---

这倒是
但是加固板不是个物品吗（）

---

是这样……那咋整？摆烂等新构建？

---

游戏内我待会看看殷钢效果怎么样

---

悲测不了
等新版本吧

---

```suggestion
  "modifier.tconstruct.temperate.reinforced": "恒温初速度",
```
```suggestion
  "modifier.tconstruct.temperate.reinforced": "恒温加强",
```